src/App.tsx

```diff
@@ -68,6 +68,14 @@ export default function App() {
         getPathDetailsStore().register(onPathDetailsChanged)
         getMapFeatureStore().register(onMapFeaturesChanged)
 
+        onQueryChanged()
+        onInfoChanged()
+        onRouteChanged()
+        onErrorChanged()
+        onMapOptionsChanged()
+        onPathDetailsChanged()
+        onMapFeaturesChanged()
```

Comment:
It can happen that the infoWithDispatch call returns **after** we initialized `setState` with the store state (so it is the initial inital state), but **before** these methods were registered for the stores above. Therefore we need to trigger the update explicitly here.